src/index.html

```diff
@@ -169,7 +169,7 @@ <h2 class="h2">Next event</h2>
                 <time datetime="2022-09-14" class="flex flex-col gap-2 items-center text-center bow-border px-4 py-2">
                   <div>Wednesday</div>
                   <div class="font-tiempos text-5xl font-semibold leading-tight">14<sup class="font-lg">th</sup> September</div>
-                  <div>6pm - 7pm, UK time - online</div>
+                  <div>FULLY BOOKED!</div>
```

Comment:
Might be nice for accessibility reasons, to use `<em class="uppercase"></em>` instead of `<div></div>`, but this is okay for now. Maybe I'll test the other thing locally sometime!